cli/command/trust/info.go

```diff
@@ -101,9 +101,50 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 	printSignatures(cli, remote, signatureRows)
 
+	roleWithSigs, err := notaryRepo.ListRoles()
+	if err != nil {
+		fmt.Fprintf(cli.Out(), "Error listing signature metadata for roles: %s\n", err)
```

Comment:
let's wrap this in a `trust.NotaryError` for now: `trust.NotaryError(ref.Name(), err)`

I'm also okay with wrapping this further with an additional message if we want, see #10 for tracking better error messages